src/CustomUrlInput.svelte

```diff
@@ -27,6 +27,12 @@
     allowPolling = false;
   });
 
+  onMount(() => {
+    if (activeUrl) {
+      fetchStyle(activeUrl);
+    }
+  });
```

Comment:
We only really wanted this to run once, so I changed `App.svelte` to set variables from the hash before mounting in  since we don't need anything to render first for that which sets us up to use `onMount` more appropriately downstream here.